activitysim/abm/tables/vehicles.py

```diff
@@ -0,0 +1,44 @@
+# ActivitySim
+# See full license in LICENSE.txt.
+import logging
+
+from activitysim.core import inject, pipeline, tracing
+
+logger = logging.getLogger(__name__)
+
+
+@inject.table()
+def vehicles(households):
```

Comment:
add typehints

Reply:
Same with above. The user would never actually directly call the `vehicles()` function to retrieve the vehicles table, so the input object types would never be of concern. I could certainly add them if you think it would help, or maybe per your comment below, a parameters list in the docstring would suffice